src/darwin/Framework/CHIP/MTROTAProviderDelegateBridge.mm

```diff
@@ -597,9 +661,17 @@ bool GetPeerNodeInfo(CommandHandler * commandHandler, const ConcreteCommandPath
                           handle.Release();
                           return;
                       }
-
+                      if (!isBDXProtocolSupported) {
+                          Commands::QueryImageResponse::Type protocolNotSupportedResponse;
+                          protocolNotSupportedResponse.status
+                              = static_cast<OTAQueryStatus>(MTROTASoftwareUpdateProviderOTAQueryStatusBusy);
```

Comment:
```suggestion
                              = static_cast<OTAQueryStatus>(MTROTASoftwareUpdateProviderOTAQueryStatusDownloadProtocolNotSupported);
```
unless I am _really_ missing something.

Reply:
ah i tested this with MTROTASoftwareUpdateProviderOTAQueryStatusDownloadProtocolNotSupported. Just moving files around caused this